.github/workflows/cmake.yml

```diff
@@ -1,11 +1,25 @@
 name: CMake
 
-on: [push,pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
I'm not sure if the previous `on` was intentional, but it was triggering this workflow twice on pushes to branches on the repo, so I set some branch limitations here.